test/controllers/account_controller_test.rb

```diff
@@ -141,6 +141,22 @@ def test_invalid_login
     assert(@request.session["user_id"])
   end
 
+  def test_email_new_password
```

Comment:
I assume this is just providing some coverage for the AccountController.  Not sure why it's buried in this PR, but I don't see any harm in it.